{{cookiecutter.project_slug}}/compose/production/django/start

```diff
@@ -6,4 +6,10 @@ set -o nounset
 
 
 python /app/manage.py collectstatic --noinput
+{%- if cookiecutter.use_whitenoise == 'y' and cookiecutter.use_compressor == 'y' %}
+if [ "${COMPRESS_ENABLED:-}" = "True" ];
```

Comment:
I'm not sure about that... As far as I can tell, this is unset in prod, and the setting default to `True` if absent. This bash `if` will never evaluate run, will it? Shouldn't it be that instead:

```suggestion
if [ "${COMPRESS_ENABLED:-True}" = "True" ];
```

Also, the way django-environ reads the environment, it [supports several values as truthy](https://django-environ.readthedocs.io/en/latest/index.html#environ.environ.Env.BOOLEAN_TRUE_STRINGS), but this supports only one.

I think I'd be OK to keep it simple with just the template branch from line 9, but I'm open to a better idea.

Reply:
Of course, I hadn't thought of that as I'm using this as an env var on my site. The issue is if compress is disabled and this line runs it causes an error, so without some check here there's no way of disabling compressor using env vars (this file would also have to change). I think the default to true should mostly work for a lot of cases though, I will change this.

Reply:
@browniebroke I think this solution using environ should cover any edge cases now. Broken link also fixed.